Content.Server/Connection/Whitelist/WhitelistPrototype.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Threading.Tasks;
+using Robust.Shared.Network;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Connection.Whitelist;
+
+[Prototype("PlayerConnectionWhitelist")]
+public sealed class PlayerConnectionWhitelistPrototype : IPrototype
```

Comment:
I need a doc comment explaining what these prototypes are and what they are used for (e.g. reference to CVar).